src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/vmware/VMWareProvider.java

```diff
@@ -245,12 +241,10 @@ public void deleteInstance(Infrastructure infrastructure, String instanceId) {
                                                     throw new RuntimeException("ERROR when powering OFF the instance : " +
                                                                                instanceId);
                                                 }
-
                                             } catch (RemoteException | InterruptedException e) {
                                                 throw new RuntimeException("ERROR when deleting VMWare instance : " +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)